.tool-versions

```diff
@@ -1,4 +1,4 @@
 golang 1.22.5
-mage 1.15.0
+mage 1.14.0
```

Comment:
I think there's still no Mac ARM build of 1.15

Reply:
🤦‍♂️